packages/runtime/lib/management-api.js

```diff
@@ -115,7 +117,11 @@ async function createManagementApi (configManager, runtimeApiClient) {
     })
 
     app.get('/logs', { websocket: true }, async (connection) => {
```

Comment:
I think we should accept the log level as a querystring parameter

Reply:
We instantiate the pino logger at the runtime startup. How we can apply the logger level to it later?